packages/astro/src/assets/vite-plugin-assets.ts

```diff
@@ -222,7 +222,7 @@ export default function assets({ fs, settings, sync, logger }: Options): vite.Pl
 					const imageMetadata = await emitESMImage(
 						id,
 						this.meta.watchMode,
-						!!settings.config.experimental.svg,
+						id.endsWith('.svg'),
```

Comment:
honestly this could just be `false` since we don't use it

Reply:
Yes exactly